client/app.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
+import AppBar from 'material-ui/AppBar'
+import axios from 'axios'
+import uuid from 'uuid/v4'
+import UsersForm from './users-form'
+
+export default class App extends Component {
+  constructor(props) {
+    super(props)
+    this.userFormStyle = { margin: '50px 0', padding: '20px 30px', textAlign: 'center' }
```

Comment:
It would be better to move this style object to the `UserForm` component directly.